extensions/amp-story/1.0/amp-story.css

```diff
@@ -273,6 +273,38 @@ amp-story-grid-layer > * {
   pointer-events: auto !important;
 }
 
+amp-story-grid-layer amp-twitter::after {
+  content: "" !important;
+  position: absolute !important;
+  height: 100% !important;
+  width: 100% !important;
+  top: 0 !important;
+  left: 0 !important;
+}
+
+amp-twitter.i-amphtml-remove-click-shield::after {
+  height: 0px !important;
+  width: 0px !important;
+}
+
+.i-amphtml-story-grid-layer-z-index-reset amp-story-grid-layer,
+.i-amphtml-story-grid-layer-z-index-reset amp-story-grid-layer * {
+  contain: none !important;
+  z-index: auto !important;
+}
+
+amp-story-page.i-amphtml-story-grid-layer-z-index-reset[active] {
+  contain: none !important;
+  z-index: 100001 !important; /* On top of i-amphtml-story-system-layer. */
+}
+
+amp-twitter.i-amphtml-increment-z-index {
+  z-index: 2 !important;
+  position: absolute !important;
+  top: 15% !important;
```

Comment:
This could probably also be more general: `top: 50%; transform: translateY(-50%);`